src/main/java/com/google/sps/servlets/FamilyServlet.java

```diff
@@ -44,10 +54,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         familyEntity.setProperty("name", familyName);
         familyEntity.setProperty("memberEmails", memberEmails);
```

Comment:
do you need to save memberEmails here, if you have a user entity that has a familyID?

You could always do a lookup like, "find all users in the family by looking up all users with familyId", yes?

Having duplicate data is unnecessary (except in rare cases for optimization, but let's worry about that when we get there, and it's not clear this is the best way to do this) and makes modifying things hardedr.